lib/routes/create_invoice/create_invoice_page.dart

```diff
@@ -209,6 +209,8 @@ class CreateInvoicePageState extends State<CreateInvoicePage> {
                 return acc.validateIncomingPayment(amount);
               }
 
+              final response = widget.lnurlWithdraw;
```

Comment:
It looks like you can use _withdrawFetchResponse variable here directly as this value is assigned to it initially.